notebooks/3_KernelMethods.ipynb

```diff
@@ -149,7 +172,7 @@
     "\\end{equation}\n",
     "one sees that $\\mathbf{\\Phi}\\mathbf{\\Phi}^T = \\mathbf{K}$: the eigenvectors of the kernel matrix make it possible to construct a set of features whose scalar product reproduces exactly the values of the kernel for the dataset. \n",
     "\n",
-    "The second equality is important because it provides a recipe to build an _approximate_ set of RKHS features for a _new_ set of points. If $\\mathbf{K}_{NN}$ indicates the matrix for the train set, and $\\mathbf{K}_{N'N}$ the matrix that contains the kernels between some new (e.g. test-set) samples and the train samples, one can compute \n",
+    "The second equality is important because it provides a recipe to build an _approximate_ set of RKHS features for a _new_ set of points. If $\\mathbf{K}_{NN}$ indicates the kernal matrix for the train set, and the matrix $\\mathbf{K}_{N'N}$ contains the kernels between some new (e.g. test-set) samples and the train samples, which means the size of $\\mathbf{K}_{N'N}$  is $n_{new\\ samples}\\times n_{train\\ sample}$,  one can compute \n",
```

Comment:
```suggestion
    "The second equality is important because it provides a recipe to build an _approximate_ set of RKHS features for a _new_ set of points. If $\\mathbf{K}_{NN}$ indicates the kernel matrix for the train set, and the matrix $\\mathbf{K}_{N'N}$ contains the kernels between some new (e.g. test-set) samples and the train samples, which means the size of $\\mathbf{K}_{N'N}$  is $n_{new\\ samples}\\times n_{train\\ sample}$,  one can compute \n",
```